.github/workflows/triton-benchmarks.yml

```
@@ -138,6 +138,24 @@ jobs:
           TAG=${{ inputs.tag || 'ci' }}
           python ../../scripts/build_report.py $REPORTS/matmul-performance.csv $REPORTS/gemm-preop-exp-triton-report.csv --benchmark gemm-preop-exp --compiler triton --param_cols "B,M,K,N" --tflops_col Triton-TFlops --hbm_col "Triton-GB/s" --tag $TAG
 
+      - name: Run Triton GEMM + PostOp (Gelu) kernel benchmark
+        if: ${{ steps.install.outcome == 'success' && !cancelled() }}
+        run: |
+          cd benchmarks/triton_kernels_benchmark
+          python gemm_postop_addmatrix_benchmark.py --reports $REPORTS
```

It's the same benchmark as in the next step, copy & paste issue?
```suggestion
          python gemm_postop_gelu_benchmark.py --reports $REPORTS
```

---

good catch, fixed